build_tools/cmake/iree_copts.cmake

```diff
@@ -299,6 +299,8 @@ list(APPEND IREE_COMMON_INCLUDE_DIRS
 # Third party: mlir-emitc
 #-------------------------------------------------------------------------------
 
+set(EMITC_ENABLE_HLO OFF)
+set(EMITC_INCLUDE_TESTS OFF)
```

Comment:
Can these be moved under `if(IREE_ENABLE_EMITC)`, similar to how the `SPIRV_CROSS_*` options are set below this only when `IREE_TARGET_BACKEND_METAL-SPIRV` is true?

Reply:
Sure!